utils/diagram_generator/web/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```

Comment:
not sure there's much point having the web version :-)

Reply:
Yeah, good point.  I just had the option enabled when I generated the shims.  Removed this dir.